docs/bom.adoc

```diff
@@ -107,6 +107,17 @@ Then include the individual modules as you see fit, by simply depending on the u
 </dependency>
 ----
 
+==== Arquillian Cube Docker Starter (Single Unified Dependency for Cube Docker in Standalone Mode)
+
+[source, xml]
+----
+<dependency>
+    <groupId>org.arquillian.cube</groupId>
+    <artifactId>arquillian-cube-docker-starter</artifactId>
+    <scope>test</scope>
+</dependency>
```

Comment:
How about sourcing these dependencies from the actual POM where it's defined? This way if we ever rename it we won't need to care about changes in the docs too.